pkg/webhook/utils.go

```diff
@@ -43,12 +43,26 @@ func (ca ConnectionAction) String() string {
 type ConnectionType string
 
 const (
-	// ConnectionTypeURI represents uri connections.
-	ConnectionTypeURI ConnectionType = "uri-v1"
-	// ConnectionTypeS3 represents s3 connections.
-	ConnectionTypeS3 ConnectionType = "s3"
-	// ConnectionTypeOCI represents oci connections.
-	ConnectionTypeOCI ConnectionType = "oci-v1"
+	// ConnectionTypeProtocolURI represents uri connections.
+	ConnectionTypeProtocolURI ConnectionType = "uri"
+	// ConnectionTypeProtocolS3 represents s3 connections.
+	ConnectionTypeProtocolS3 ConnectionType = "s3"
+	// ConnectionTypeProtocolOCI represents oci connections.
+	ConnectionTypeProtocolOCI ConnectionType = "oci"
+)
+
+// Deprecated: ConnectionTypeRef constants are deprecated in favor of ConnectionTypeProtocol constants.
```

Comment:
i think you just need to change it to 

`// ConnectionTypeRef constants are deprecated in favor of ConnectionTypeProtocol constants.`

then the statickcheck nolint can be removed